tests/test_views.py

```diff
@@ -1,21 +1,87 @@
 """
-Tests for the frontend.
+Unit tests for the frontend code.
 """
 
+import os
+import tempfile
 import unittest
 
-import ga4gh.server as server
+import ga4gh.backends
 import ga4gh.cli
+import ga4gh.protocol as protocol
+import ga4gh.server as server
+
+
+class MockBackend(ga4gh.backends.Backend):
+    """
+    A mock Backend class for testing.
+    """
+    def __init__(self, dataDir):
+        self._dataDir = dataDir
+
+    def searchVariants(self, request):
+        response = protocol.GASearchVariantsResponse()
```

Comment:
We should just be able to return `protocol.GASearchVariantsResponse()` as  the default value for `variants` is the empty list.